governance/xc_admin/packages/xc_admin_common/src/chains.ts

```diff
@@ -67,6 +67,7 @@ export const RECEIVER_CHAINS = {
   parallel: 60048,
   iota: 60049,
   flow_previewnet: 60050,
+  starknet: 60051,
```

Comment:
These are probably outdated now. Let's rebase it and merge it soon (so we don't use it)

Reply:
Rebased.